README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Project Documentation
+
+- Join us on [Open session](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions) and see [Meeting minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit)
+- For setting up the project locally watch [this video](https://youtu.be/_b2RQGbYN9w)
+- Checkout the video of [GSOC](https://youtu.be/3A746GppZ0Y) here
+- Have questions in mind feel free to ask on [#question stream](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Questions)
+
+For a more complete information on OpenSourcePrograms backend project, please go to [OpenSourcePrograms Backend Wiki page](https://github.com/anitab-org/open-source-programs-backend/wiki)
```

Comment:
```suggestion
For more complete information on the Open Source Programs backend project, please go to [OpenSourcePrograms Backend Wiki page](https://github.com/anitab-org/open-source-programs-backend/wiki).
```

Reply:
You can also write this:
`For more information, see [open-source-programs-backend wiki](https://github.com/anitab-org/open-source-programs-backend/wiki).`

Reply:
Thanks for pointing that